data/announcements/scheduled.yaml

```diff
@@ -134,21 +134,30 @@ announcements:
 - name: Freezing k8s.gcr.io - Before
   startTime: 2023-02-20T00:00:00 # Added in https://github.com/kubernetes/website/pull/39575
                                  # This should run before and after Kubecon EU 2023
-  endTime: 2023-03-26T00:00:00
+  endTime: 2023-03-10T00:00:00
   style: "background: #c70202"
   title: "Legacy k8s.gcr.io container image registry will be frozen in early April 2023"
   message: |
     k8s.gcr.io image registry will be frozen from the 3rd of April 2023.<br/>
     Images for Kubernetes 1.27 will not be available in the k8s.gcr.io image registry.<br/>
     Please read our [announcement](/blog/2023/02/06/k8s-gcr-io-freeze-announcement/) for more details.
 
-- name: Freezing k8s.gcr.io - After
-  startTime: 2023-04-05T00:00:00 # Added in https://github.com/kubernetes/website/pull/39575
-                                 # This should run before and after Kubecon EU 2023
+- name: Redirecting k8s.gcr.io - Before
+  startTime: 2023-03-10T00:00:00 # This should run before and after Kubecon EU 2023
+  endTime: 2023-03-26T00:00:00
+  style: "background: #c70202"
+  title: Legacy k8s.gcr.io container image registry will be redirected to registry.k8s.io
+  message: |
+    k8s.gcr.io image registry will be redirected to registry.k8s.io on Monday March 20th.<br>
+    All images available in k8s.gcr.io are available at registry.k8s.io.</br>
+    Please read our [announcement](blog/2023/03/10/image-registry-redirect/) for more details.
+
+- name: Redirecting k8s.gcr.io - After
+  startTime: 2023-04-05T00:00:00 # This should run before and after Kubecon EU 2023
   endTime: 2023-05-31T00:00:00
   style: "background: #c70202"
-  title: Legacy k8s.gcr.io container image registry was frozen on the 3rd of April 2023
+  title: Legacy k8s.gcr.io container image registry will be redirected to registry.k8s.io
   message: |
-    k8s.gcr.io image registry has been frozen on the 3rd of April 2023.<br/>
-    Images for Kubernetes 1.27 are not available in the k8s.gcr.io image registry.<br/>
-    Please read our [announcement](/blog/2023/02/06/k8s-gcr-io-freeze-announcement/) for more details.
+    k8s.gcr.io image registry will be redirected to registry.k8s.io on Monday March 20th.<br>
+    All images available in k8s.gcr.io are available at registry.k8s.io.</br>
+    Please read our [announcement](blog/2023/03/10/image-registry-redirect/) for more details.
```

Comment:
```suggestion
    Please read our [announcement](/blog/2023/03/10/image-registry-redirect/) for more details.
```

Reply:
thankfully the current link appears to work in my browser at least, even if non-idiomatic. phew. https://github.com/kubernetes/website/pull/39918 contains this and your other suggestions

Reply:
It works for https://k8s.io/ but not for other pages.